### PR TITLE
Database set up

### LINKED ISSUE #4: Set up database

### PR DESCRIPTION
**Pull request by zeinabfarag-zz**
to test:
run npm install

 make sure mongodb is installed on your computer.
 run "mongod" in terminal.

in another terminal cd into the server folder and "npm run dev" 

should have following in terminal:
Database connection successful

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -36,4 +40,19 @@ app.use(function(err, req, res, next) {
...
+  })
+  .catch(err => {
+    console.error("Database connection error:", err);
+  });
+
+//listen to server
+app.listen(port, () => {
+  console.log(`Express server running on port ${port}`);
+});
+
```
Comment (aeciorc) [inline server/app.js]: I'd move this to a separate file and import it to keep this cleaner

Comment (aeciorc) [inline server/app.js]: This is actually already done on `bin/www`

File: server/app.js [hunk 2]
```diff
@@ -7,9 +7,13 @@ const logger = require("morgan");
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 
+const mongoose = require("mongoose");
+const uri = "mongodb://localhost/kanban";
```
Comment (aeciorc) [inline server/app.js]: I'd put this uri on .env

**Review by aeciorc — CHANGES_REQUESTED** [review]
Looks good, a few comments below

**Comment by zeinabfarag-zz** [comment]
Thanks for the review @aeciorc please let me know if I need to do any other modifications

**Review by aeciorc — APPROVED** [review]
Good job 👍

**Pull request merged**